src/cli/ionic-config.ts

```diff
@@ -0,0 +1,45 @@
+import { readJSON, writeJSON, mkdirp } from '@ionic/utils-fs';
```

Comment:
I don't think you can import this without adding it to `dependencies`. Given our zero dependencies policy you may just need to copy the code from the ionic repo to the stencil repo.

Reply:
The same applies to `@ionic/utils-terminal` and `@ionic/utils-subprocess`.